src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -143,3 +150,46 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
         except Exception as e:
             logger.error(f"Couldn't fetch information for table {full_name} : {e}")
             return None
+
+    def migrate_tables(self):
+        if self._workspace_config.database_to_catalog_mapping:
+            for table in self._fetch_tables():
+                target_catalog = self._workspace_config.database_to_catalog_mapping.get(
+                    table.database, self._workspace_config.default_catalog
+                )
+                self._migrate_table(target_catalog, table)
+        else:
+            for table in self._fetch_tables():
+                self._migrate_table(self._workspace_config.default_catalog, table)
+
+    def _fetch_tables(self):
+        try:
+            tables = []
+            for row in self._backend.fetch(
+                f"SELECT * FROM hive_metastore.{self._workspace_config.inventory_database}.tables"
+            ):
+                tables.append(Table(*row))
+            logger.debug(f"Found {len(tables)} tables to migrate")
+            return tables
+        except Exception as e:
+            logger.error(f"Could not query inventory table : {e}")
+            raise e
+
+    def _migrate_table(self, target_catalog, table):
+        try:
+            sql = table.uc_create_sql(target_catalog)
+            logger.debug(f"Migrating table {table.key} to using SQL query: {sql}")
+            if table.object_type == "EXTERNAL":
+                sync_status = SyncStatus(*next(iter(self._backend.fetch(sql))))
+                if sync_status.status_code != "SUCCESS":
+                    logger.error(
+                        f"Could not sync external table {table.key} to {target_catalog}.{table.database} "
+                        f"because: {sync_status.status_code} {sync_status.description}"
+                    )
+                else:
+                    self._backend.execute(table.sql_alter(target_catalog))
```

Comment:
```suggestion
                if sync_status.status_code != "SUCCESS":
                    logger.error(
                        f"Could not sync external table {table.key} to {target_catalog}.{table.database} "
                        f"because: {sync_status.status_code} {sync_status.description}"
                    )
                    continue
                self._backend.execute(table.sql_alter(target_catalog))
```
isn't it more readable? ;)